x-pack/plugins/monitoring/server/routes/api/v1/alerts/enable.ts

```diff
@@ -82,23 +81,15 @@ export function enableAlertsRoute(server: MonitoringCore, npRoute: RouteDependen
           },
         ];
 
-        let createdAlerts: Array<SanitizedRule<RuleTypeParams>> = [];
-        const disabledWatcherClusterAlerts = await disableWatcherClusterAlerts(
-          npRoute.cluster.asScoped(request).asCurrentUser,
-          npRoute.logger
+        const createdAlerts: Array<SanitizedRule<RuleTypeParams>> = await Promise.all(
+          alerts.map((alert) => alert.createIfDoesNotExist(rulesClient, actionsClient, actions))
```

Comment:
Q: I know that we are checking if alerts exist before we create them but should we show the modal in case the default rules are already enabled? I was thinking as an idea that after clicking on `Create default rules` when they are created already using the first modal on the screen we can show a message that the default rules are already created (nothing blocking just a suggestion)

Reply:
I can imagine a case where maybe they have been created but one or more was deleted? Not sure how closely we check the status.

I think I'll pass on this change for now.